deps/rabbit/test/rabbit_stream_queue_SUITE.erl

```diff
@@ -190,6 +190,13 @@ init_per_group(cluster_size_3_parallel = Group, Config) ->
         _ ->
             init_per_group1(Group, Config)
     end;
+init_per_group(unclustered_size_3_4 = Group, Config) ->
+    case rabbit_ct_helpers:is_mixed_versions() of
```

Comment:
Is is incompatible with mixed verion cluster in general, or just incompatible with 3.12.x because it lacks something for the tests?

Reply:
it has assertions on behaviour that 3.12 doesn't have such as removal of stream coordinator members as part of `forget_cluster_node`

Reply:
Then the condition makes sense. Thanks for clarifying.

Reply:
Isn’t there another way to check the actual condition you mention, “lack of stream coordinator members removal as part of `forget_cluster_node`”?

Because the `is_mixed_version()` will remain true even after the old version in the test is set to RabbitMQ 15.0.

Reply:
I managed to make the test mixed compat.